Rakefile

```diff
@@ -1,22 +1,41 @@
 task :dev do
-  sh "bundle exec rackup -o 0.0.0.0 -p 8080 --env development"
+  sh "bundle exec rackup -p 8080 --env development"
 end
 
 task :prod do
   sh "bundle exec rackup -p 8080 --env production"
 end
 
-task :hosted_prod_test do
-  sh("FASTLANE_CI_ERB_CLIENT=1 DEBUG=1 FASTLANE_CI_DISABLE_REMOTE_STATUS_UPDATE=1 FASTLANE_CI_DISABLE_PUSHES=1 FASTLANE_CI_SKIP_RESTARTING_PENDING_WORK=1 bundle exec rackup --host 0.0.0.0 -p 8080 --env development")
-end
-
-task :devbootstrap do
+task :dev_bootstrap do
   sh("bundle install")
   sh("brew install node") unless sh("npm -v")
   sh("npm install")
   sh("ln -sf ../../.pre-commit .git/hooks/pre-commit")
 end
 
+# It seems like mapping ports in Docker requires --host which can break hosting in VMs
+# So we'll just keep some docker-specific tasks down here
+namespace :docker do
+  desc "Run in a Docker container in production mode"
+  task :prod do
+    sh "bundle exec rackup --host 0.0.0.0 -p 8080 --env production"
```

Comment:
Maybe I'm just confused here, but didn't you want the 0.0.0.0 to not be in `docker`?

Reply:
Docker needs 0.0.0.0 for the host, while VMs shouldn't have it. So, if we're running on local or in a VM, we can just use `prod`

Reply:
You aren't confused, I was 🙃